src/main/java/codewars/tasks/java/Block.java

```diff
@@ -0,0 +1,40 @@
+package codewars.tasks.java;
+
+public class Block {
+
+        private int width;
+        private int length;
+        private int height;
+
+        Block(int[] measurements) {
+            if(measurements.length == 3) {
+                this.width = measurements[0];
+                this.length = measurements[1];
+                this.height = measurements[2];
+            }
+        }
+
+        public int getWidth() {
```

Comment:
Actually everything is good. But get method for width, length, height are extra.
You can write in next way:

`
public int getVolume() {
            int volume = this.width * this.length * this.height;
            System.out.println("Block volume with params "+ length +" "+width+" "+height+" is "+volume);
            return volume;
        }
`